packages/internal/src/build/api.ts

```diff
@@ -137,7 +148,9 @@ export const transpileApi = (files: string[], options = {}) => {
     format: 'cjs',
     bundle: false,
     outdir: rwjsPaths.api.dist,
-    sourcemap: 'external', // figure out what's best during development.
+    // inline sourcemaps are preferrable while developing
+    // fewer opportunities for inconsitent states
+    sourcemap: 'inline',
```

Comment:
Do you think we could mark them as external when we build for production? Is that a concern that we should have? Surely people still want sourcemaps when sending exception reports to Sentry and friends? Or do they upload them in some complicated build pipeline?

To rephrase my question. I love sourcemaps, and I think they're valuable, but should we concerned about size when it comes to serverless?

Reply:
To be honest I don't really know what end users do in production :). But I understand how filesize might become an issue, and after a few hours of tinkering, I did manage to get external sourcemaps working with the Debugger, so I am going to follow your suggestion here. I just pushed a new commit to change this.

Note: You previously had `sourcemap:'external'`, which doesn't add the `sourceMappingURL` comment to the file. So I now changed it to `sourcemap: true`.

Reply:
Ah! Nice to know, and thanks for the comment in the code!